libs/community/langchain_community/vectorstores/azure_cosmos_db_no_sql.py

```diff
@@ -0,0 +1,276 @@
+import uuid
+from typing import Any, Dict, Iterable, List, Optional, Tuple, Type
+
+import numpy as np
+from azure.cosmos.container import ContainerProxy
+from azure.cosmos.cosmos_client import CosmosClient
+from azure.cosmos.database import DatabaseProxy
+from langchain_core.documents import Document
+from langchain_core.embeddings import Embeddings
+from langchain_core.vectorstores import VST, VectorStore
+
+from langchain_community.vectorstores.utils import maximal_marginal_relevance
+
+
+# You can read more about vector search using AzureCosmosDBNoSQL here.
+# https://aka.ms/CosmosVectorSearch
+class AzureCosmosDBNoSqlVectorSearch(VectorStore):
+    cosmos_client: CosmosClient
+    database: DatabaseProxy
+    container: ContainerProxy
+    database_name: str
+    container_name: str
+    partition_key: str
+    embedding: Embeddings
+    vector_embedding_policy: Dict[str, Any]
+    indexing_policy: Dict[str, Any]
+    cosmos_container_properties: Dict[str, Any]
+    embedding_key: str
```

Comment:
VectorStore isn't a pydantic class, so currently these are class attributes (not instance attributes)

Reply:
Updated